tests/integration/test_bayesian_optimization.py

```diff
@@ -73,9 +68,8 @@ def build_model(data: Dataset) -> GaussianProcessRegression:
     best_y = dataset.observations[arg_min_idx]
     best_x = dataset.query_points[arg_min_idx]
 
-    argmin = tf.cast(BRANIN_GLOBAL_ARGMIN, tf.float64)
-    relative_argmin_err = tf.abs((best_x - argmin) / argmin)
+    relative_minimizer_err = tf.abs((best_x - BRANIN_MINIMIZERS) / BRANIN_MINIMIZERS)
```

Comment:
Branin has several minimizers, we would care here about best_x being close to one of them, not all of them, right?

Reply:
indeed, that's what we're doing. btw I've just renamed things and changed dtypes